alfredxml/type.go

```diff
@@ -152,6 +155,20 @@ type ClientNum struct {
 	N       int `xml:",chardata"`
 }
 
+// S2nDevice is used for xml coding
+type S2nDevice struct {
+	XMLName    xml.Name
+	Name       string  `xml:"name,omitempty"`
+	MacAddress string  `xml:"mac,omitempty"`
+	Firmware   string  `xml:"fw,omitempty"`
+	Hardware   string  `xml:"hw,omitempty"`
+	RemoteMac  string  `xml:"removemac,omitempty"`
+	Signal     int     `xml:"signal,omitempty"`
+	Capacity   float32 `xml:"capacity,omitempty"`
```

Comment:
`float64` might be more convenient here, since all functions from package `math` are defined for `float64`s.